src/main/java/org/jabref/JabRefExecutorService.java

```diff
@@ -87,6 +87,20 @@ public boolean executeAndWait(Callable command) {
         }
     }
 
+    /**
+     * This method allows to execute a callable task that provides a return value after the calculation is done.
+     * @param command The task to execute.
+     * @return A Future object that provides the returning value.
+     */
+    public <T> Future<T> executeAndReturn(Callable<T> command) {
+        if (command == null) {
+            LOGGER.debug("Received null as command for execution");
+            return null;
```

Comment:
`null` should never be used. Use Optionals. (See http://java.by-comparison.com/ and effective java).

In the concrete case, use `Objects.requireNonNull()`.